lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -2138,6 +2153,22 @@ void SessionState::update_dropped_data_metrics(
       LABEL_APN, apn.c_str(), LABEL_MSISDN, msisdn.c_str(), LABEL_DIRECTION,
       DIRECTION_DOWN);
 }
+/*
+ * If UPF received session version doesn't match with SMF local
+ * session version no, then we continue to resend till SESSION_THROTTLE_CNT
+ * reaches
+ */
+bool SessionState::inc_throttle() {
+  if (throttle_count_++ < SESSION_THROTTLE_CNT) {
```

Comment:
`return throttle_count_++ < SESSION_THROTTLE_CNT` ?

Reply:
I was impressed with your code optimization suggestions.

Reply:
Don't think it optimizes anything :) But just cleaner to read.